pypst/show.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+
+from pypst.content import Content
+from pypst.functional import Functional
+from pypst.renderable import Renderable
+from pypst.utils import render_code
+
+
+@dataclass
+class Show:
```

Comment:
If we change the name of `Set`, I'd change this one, too, to `ShowRule` for consistency. Otherwise, I'd keep it as is.